src/resizable.directive.ts

```diff
@@ -785,7 +795,10 @@ export class ResizableDirective implements OnInit, OnChanges, OnDestroy {
    * @hidden
    */
   ngOnDestroy(): void {
-    this.renderer.setStyle(document.body, 'cursor', '');
+    // browser check for angular universal, because it doesn't know what document is
+    if (isPlatformBrowser(this.platformId)) {
+      this.renderer.setStyle(document.body, 'cursor', '');
```

Comment:
I think this is the only place that requires the check, as this commit added the line in a recent version which is why it's only just broken: https://github.com/mattlewis92/angular-resizable-element/commit/3ee7e6259f360900e9642843687a9c9fade8b13a

Reply:
The only place I actually received the error was in the OnDestroy function. But I put it everywhere normally just to be sure, as I don‘t know your component so well.